tom_demoapp/apps.py

```diff
@@ -29,16 +30,17 @@ def nav_items(self):
         be included in the navbar. The `position` key, if included, should be either "left" or "right" to specify which
         side of the navbar the partial should be included on. If not included, a right side nav item is assumed.
         """
-        return [{'partial': 'tom_demoapp/partials/navbar_demo.html', 'position': 'right'},
-                {'partial': 'tom_demoapp/partials/navbar_list_demo.html'}]
+        # TODO: These filenames probably don't need 'demo' in them b/c they're namespaced in the app folder
```

Comment:
I don't think this is a good idea. I think these having descriptive partials with the app name included will be very useful for making sure a user always knows what html they are editing.

Reply:
I don't feel strongly about this